sklearn/feature_extraction/text.py

```diff
@@ -425,7 +425,7 @@ def _check_stop_words_consistency(self, stop_words, preprocess, tokenize):
     def build_analyzer(self):
         """Return a callable to process input data.
 
-        The callable handles that handles preprocessing, tokenization, and
+        The callable handles preprocessing, tokenization, and
         n-grams generation.
```

Comment:
Minor nit: I'd prefer to use as much horizontal line space as possible (79 characters per pep8).
```suggestion
        The callable handles preprocessing, tokenization, and n-grams
        generation.
```

Reply:
it is even 88 (with our black standard) and it would fit on a single line